packages/ckb/src/constants/index.ts

```diff
@@ -294,6 +276,33 @@ const MainnetInfo = {
   } as CKBComponents.Script,
 };
 
+export const COMPATIBLE_XUDT_TYPE_SCRIPTS: CKBComponents.Script[] = [
+  // RUSD Mainnet
+  {
+    codeHash: '0x26a33e0815888a4a0614a0b7d09fa951e0993ff21e55905510104a0b1312032b',
+    hashType: 'type',
+    args: '',
+  },
+  // RUSD Testnet
+  {
+    codeHash: '0x1142755a044bf2ee358cba9f2da187ce928c91cd4dc8692ded0337efa677d21a',
+    hashType: 'type',
+    args: '',
+  },
+  // USDI Mainnet
+  {
+    codeHash: '0xbfa35a9c38a676682b65ade8f02be164d48632281477e36f8dc2f41f79e56bfc',
+    hashType: 'type',
+    args: '',
+  },
+  // USDI Testnet
+  {
+    codeHash: '0xcc9dc33ef234e14bc788c43a4848556a5fb16401a04662fc55db9bb201987037',
+    hashType: 'type',
+    args: '',
+  },
```

Comment:
https://github.com/utxostack/typeid-contract-cell-deps/blob/main/compatible-udt.json does not yet include USDI.

Are you going to update it later and verify the process designed after this PR?
Or what is the plan?

Reply:
I am adding USDI to [compatible-udt.json](https://github.com/utxostack/typeid-contract-cell-deps/blob/main/compatible-udt.json) now.

Reply:
https://github.com/utxostack/typeid-contract-cell-deps/pull/19

Reply:
Tested with USDI in https://github.com/utxostack/rgbpp-sdk/pull/297/commits/47c7e8d642892c267c15f8ff122e5e0e010c5e30